backend/src/main/java/com/bakdata/conquery/util/io/ClassPathURLStreamHandlerProvider.java

```diff
@@ -0,0 +1,22 @@
+package com.bakdata.conquery.util.io;
+
+import java.net.URL;
+import java.net.URLStreamHandler;
+import java.net.spi.URLStreamHandlerProvider;
+
+import com.google.auto.service.AutoService;
+
+/**
+ * Allows parsing URLs that start with "classpath://" to access resources on the classpath. The actual access to the resource is done using
+ * {@link ClassPathHandler#openConnection(URL)} or through the {@link ClassPathFileSystemProvider}, when the URL is converted to a URI.
+ */
+@AutoService(URLStreamHandlerProvider.class)
+public class ClassPathURLStreamHandlerProvider extends URLStreamHandlerProvider {
```

Comment:
brauchst du die?

Reply:
Ja ich benutze die URLs Mapper Templates damit ich in den tests ohne Probleme auf den classpath verweisen kann